text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/tag/Tag.java

```diff
@@ -216,5 +217,21 @@ default boolean isFalse() {
         return OptionalDouble.empty();
       }
     }
+
+    /**
+     * Try to return this argument as a {@code char}.
+     *
+     * <p>The optional will only be present if the value is a valid char.</p>
+     *
+     * @return an optional providing the value of this argument as an integer
+     * @since 4.10.0
+     */
+    default @NotNull Optional<Character> asChar() {
```

Comment:
Is this used anywhere? Why was this added?

Reply:
I tested some stuff but I don't know why I added this. I'll remove it.